### ISSUE TITLE
Bibtex entry

### ISSUE THREAD

**Issue by sebastianJaeger** [ComPWA/pycompwa#89]
To properly cite the pycompwa package, it would be useful to have some bibtex entry on the website that can just be copypasted. Ideally, it would also automatically update its information, for example based on the setup file.

**Comment by spflueger** [comment]
Good point. Ideally we had zenodo for that, but that is currently not working.

Since we plan to freeze the pycompwa project anyways and move onto tensorwaves, I think we can just keep it simple and make a bibtex entry in which the version is managed by the user.

**Comment by redeboer** [comment]
Yeah, see https://github.com/ComPWA/pycompwa/issues/18

**Comment by redeboer** [comment]
```bibtex
@misc{fritschPyComPWACommon2020,
  title = {(py){{ComPWA}} – the {{Common Partial Wave Analysis}} Framework},
  author = {Fritsch, Miriam and Goetzen, Klaus and Gradl, Wolfgang and Michel, Mathias and Peters, Klaus and Pfl{\"u}ger, Stefan and Pitka, Andreas and Weidenkaff, Peter and Zhang, Jingqing and de Boer, Remco},
  year = {2020},
  month = apr,
  day = 22,
  url = {https://github.com/ComPWA/pycompwa/tree/817e80fbf19a1da59453813cf4ff336d4c76acc4},
}
```
Where you need to point to the correct commit (and date?).
@sebastianJaeger could you make a PR for some example bibcit file like this? For reference, you can use the [old Zenodo entry](https://zenodo.org/record/3479482).
